cypress/e2e/registration.cy.js

```diff
@@ -1,11 +1,125 @@
 /// <reference types='cypress' />
 
+const { generateUser } = require("../support/generate");
+const { randomGender } = require("../support/gender");
+const { randomHobby } = require("../support/hobby");
+const { randomState } = require("../support/state");
+
 describe('Student Registration page', () => {
-  before(() => {
-    
+  beforeEach(() => {
+    cy.visit('/')
   });
 
-  it('', () => {
+it('should be valid page', () => {
```

Comment:
e2e test is about some automated flow. no need to add a separate test that will check the presence of some field, and in another test just type into this field, because cypress won't be able to type smth if the field doesn't exist.